tests/shredder/test_config.py

```diff
@@ -0,0 +1,111 @@
+from multiprocessing.pool import ThreadPool
+
+from google.cloud import bigquery
+
+from bigquery_etl.shredder.config import DeleteSource, DeleteTarget, find_glean_targets
+
+
+class FakeClient:
+    def list_datasets(self, project):
+        return [
+            bigquery.DatasetReference(project, f"{app}_{suffix}")
+            for app in ["fenix", "focus_android"]
+            for suffix in ["stable", "derived"]
+        ]
+
+    def list_tables(self, dataset_ref):
+        labels = {}
+        if dataset_ref.dataset_id.endswith("stable"):
+            table_ids = ["metrics_v1", "deletion_request_v1"]
```

Comment:
For better test coverage it'd be good to have a table that starts with `migration` and verify it's excluded (i.e. no change to expected output).